intera_interface/src/intera_motion_interface/interaction_options.py

```diff
@@ -239,6 +245,29 @@ def set_interaction_control_mode(self, interaction_mode = None):
                 rospy.logerr('The number of interaction_control_mode elements must be 1 or %d',
                              self.n_dim_cart)
 
+    def set_disable_damping_in_force_control(self, disable_damping_in_force_control = None):
+        """
+        @param disable_damping_in_force_control:
+          - None:  set it to False by default
+          - [bool]: copy the element.
+        """
+        if disable_damping_in_force_control is None:
+            self.set_disable_damping_in_force_control(self.default_disable_damping_in_force_control) # default value
+        else:
+            self._data.disable_damping_in_force_control = disable_damping_in_force_control
+
+    def set_disable_reference_resetting(self, disable_reference_resetting = None):
+        """
+        @param disable_reference_resetting:
+          - None:  set it to False by default
+          - [bool]: copy the element.
+        """
+        if disable_reference_resetting is None:
+            self.set_disable_reference_resetting(self.disable_reference_resetting) # default value
```

Comment:
- [ ] This should be `...resetting(self.default_disable_reference_resetting)`, not `self.disable_refe...`, right?